symengine/flint_wrapper.h

```diff
@@ -627,6 +667,13 @@ class fmpz_poly_wrapper
         return fmpz_poly_divrem(*q.get_fmpz_poly_t(), *r.get_fmpz_poly_t(),
                                 *get_fmpz_poly_t(), *b.get_fmpz_poly_t());
     }
+
+    fmpz_poly_factor_wrapper factors() const
+    {
+        fmpz_poly_factor_wrapper r;
+        fmpz_poly_factor(r.get_fmpz_poly_factor_t(), poly);
```

Comment:
A test to check for error if version<=2.5.2 and the current tests only for version>2.5.2. Is this what you meant by updating tests @isuruf ?